vita3k/renderer/include/renderer/state.h

```diff
@@ -88,6 +89,9 @@ struct State {
     void set_surface_sync_state(bool disable) {
         disable_surface_sync = disable;
     }
+    void set_display_area(bool disable) {
```

Comment:
this function name is not really appropriate, something like set_stretch_display would be better.

Reply:
exactly